### PR TITLE
Issue #5. Basic HTML Structure of the homepage

### PR DESCRIPTION
**Pull request by nhiieL**

### CODE REVIEW

File: static/index.html [hunk 1]
```diff
@@ -0,0 +1,39 @@
+<!DOCTYPE html>
+<html lang = "en">
+<head>
+ <meta charset="utf-8">
+ <meta http-equiv="X-UA-Compatible" content="IE=edge">
+ <meta name="viewport" content="width=device-width, initial-scale=1">
+ <title>Drupalcamp Cebu 2018</title>
+
+ <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/css/bootstrap.min.css" integrity="sha384-BVYiiSIFeK1dGmJRAkycuHAHRg32OmUcww7on3RYdg4Va+PmSTsz/K68vbdEjh4u" crossorigin="anonymous">
+ <link rel="stylesheet" href="/assets/style.css">
```
Comment (Luukyb) [inline static/index.html]: Please make sure the css is working. Change the background color for example on your SASS, and it should change in your webpage.

File: static/index.html [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+ <link href="https://fonts.googleapis.com/css?family=Muli:200,400|Saira:400,700" rel="stylesheet">
```
Comment (Luukyb) [inline static/index.html]: This is not the fonts we are using. Please use Google fonts to configure the new site.

File: static/index.html [hunk 3]
```diff
@@ -7,8 +7,9 @@
  <title>Drupalcamp Cebu 2018</title>
 
  <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/css/bootstrap.min.css" integrity="sha384-BVYiiSIFeK1dGmJRAkycuHAHRg32OmUcww7on3RYdg4Va+PmSTsz/K68vbdEjh4u" crossorigin="anonymous">
- <link rel="stylesheet" href="/assets/style.css">
- <link href="https://fonts.googleapis.com/css?family=Muli:200,400|Saira:400,700" rel="stylesheet">
+ <link rel="stylesheet" href="./css/style.css">
```
Comment (Luukyb) [inline static/index.html]: Why is there a dot?

File: static/index.html [hunk 4]
```diff
@@ -7,8 +7,9 @@
...
+<link href="https://fonts.googleapis.com/css?family=Open+Sans" rel="stylesheet">
```
Comment (Luukyb) [inline static/index.html]: I believe google fonts can be added as one line, not two. One line = One request. Faster to download once, than two times.

File: static/index.html (hunk 3 above)
Comment (marychris) [reply static/index.html]: Try removing the dot and do a restest :)

File: static/index.html (hunk 4 above)
Comment (marychris) [reply static/index.html]: Use this instead:
<link href="https://fonts.googleapis.com/css?family=Merriweather:700|Open+Sans" rel="stylesheet">

**Pull request merged**